src/main/java/net/snowflake/client/log/JDK14Logger.java

```diff
@@ -73,7 +73,13 @@ public boolean isWarnEnabled()
 
   public void debug(String msg)
   {
-    logInternal(Level.FINE, msg);
+    logInternal(Level.FINE, msg, true);
+  }
+
+  // Please don't use this function except you know what you are doing
```

Comment:
I think a more specific comment would be helpful here. Such as "This function is used to display unmasked, potentially sensitive log information for internal regression testing purposes. Do not use otherwise" or something along those lines

Reply:
Sure, will change it!